File: AngularKajke/Models/ModelClasses/Question.cs
namespace AngularKajke.Models.ModelClasses
{
  public class Question
  {
    public string Query { get; set; }
    public short QuestionNumber { get; set; }
    public AnswerType AnswerType { get; set; }
    public string[] Answers { get; set; }
    public bool IsOptional { get; set; }
  }
}

File: AngularKajke/src/app/shared/Models/question.ts
import { AnswerType } from './answer-type.enum';

export class Question {
  public constructor(init?: Partial<Question>) {
    Object.assign(this, init);
  }
  QuestionErrors?: string;
  Query?: string;
  QuestionNumber?: number;
  AnswerType?: AnswerType;
  Answers?: string[];
  IsOptional?: boolean;
}
